sonic_installer/bootloader/bootloader.py

```diff
@@ -75,6 +75,10 @@ def supports_package_migration(self, image):
         """tells if the image supports package migration"""
         return True
 
+    def verify_image_sign(self, image_path):
+        """verify image signature is valid"""
+        return True
```

Comment:
>True [](http://example.com/codeflow?start=15&length=4)

I do not understand the semantics of the return value, and why using `True` as default value. A unsecure normal image is considered not signed image, right?

Reply:
unsecured normal image is considered not signed, yes

Reply:
@ycoheNvidia try/except would be more appropriate? 

```
        echo_and_log("Verifing image {} signature...".format(binary_image_version))
        try : 
               if not bootloader.verify_image_sign(image_path):
                        echo_and_log('Error: Failed verify image signature', LOG_ERR)
                        raise click.Abort()
                else:
                       echo_and_log('Verification successful')
           except AttributeError:
                     echo_and_log("Skip Verifing image {} signature...Not signed image".format(binary_image_version))
                      pass
                     
 ```

Reply:
I am not sure which lines you are referring to @prgeor, can you please elaborate?

Reply:
@ycoheNvidia @liat-grozovik instead of defining verify_image_sign() as NOP function in the bootloader.py can we try the except/catch block I showed above? So that if a bootloader does not implement verify_image_sign(), it won't crash here [https://github.com/sonic-net/sonic-utilities/blob/master/sonic_installer/main.py#L582](https://github.com/sonic-net/sonic-utilities/blob/master/sonic_installer/main.py#L582)